<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;images/9.jpg&quot;);background-repeat: no-repeat, repeat;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:0;color:#fff;text-aligne:center;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>SELECT A GAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1to10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ffa502;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>1 to 10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1to100">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ffa502;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>1 to 100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_500">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ffa502;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>-500 to 500</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
